client/layout/index.jsx

```diff
@@ -161,6 +162,10 @@ Layout = React.createClass( {
 					{ this.renderWelcome() }
 					{ this.renderEmailVerificationNotice() }
 					<GlobalNotices id="notices" notices={ notices.list } forcePinned={ 'post' === this.props.section } />
+					<DomainDiagnostic domain='donaldjdrumpf.com' />
```

Comment:
Just an example, this will be mounted elsewhere. If you have any ideas where -> please let me know :)